[IMP] POS: customer display config and use update (#14000)
odoo/documentation: content/applications/sales/point_of_sale/shop/customer_display.rst

   :alt: customer screen
   :scale: 50 %

.. note::
   Both the customer display and POS display should be at least 6 inches. However, a larger screen
   is recommended for improved end-user readability.

Configuration
=============

Depending on the POS setup, feature can be displayed :ref:`directly on a secondary screen
<customer_display/local>` connected via USB-C or HDMI, :ref:`on another device remotely
<customer_display/remote>`, or :ref:`on a screen connected through an IoT system
<customer_display/iot>`.

To activate the feature, follow these steps:

#. Navigate to the :ref:`POS settings <configuration/settings>`.
#. Scroll down to the :guilabel:`Connected Devices` section.
#. Open the dropdown menu under the :guilabel:`Customer Display` section and select one of the
   following options:

   - :guilabel:`None`: To disable the customer display.
   - :guilabel:`The same device`: To use the secondary screen connected with an HDMI or USB-C cable.
   - :guilabel:`Another device`: To connect to a remote device connected to the database.
   - :guilabel:`IOT-connected screen` for displays connected via an :doc:`IoT system
     <../../../general/iot>`.

> For displays connected using an :doc:`IoT system <../../../general/iot>`:

Reviewer: On Odoo 18, on a trial DB (not the runbot), if you configure everything for the first time, you have to:
- Enable IoT Box
- Save
- Select an already "configured" Connected Display (insert a ref here or explain?) 
- Then, and only then, you can select the "An IOT-connected screen" under Customer Display.

Otherwise, you get this error:
<img width="945" height="438" alt="image" src="https://github.com/user-attachments/assets/d95c17bf-4804-43cb-a867-f4260af1c315" />

#. From the same section, tick the :guilabel:`IoT Box` checkbox to activate the IoT system in POS.
#. Select the IoT-connected screen from the dropdown menu in the :guilabel:`Customer Display` field.

Reaching the customer display
=============================

.. _customer_display/local:

Local
-----

To open the customer display on a second screen connected to a POS system using an HDMI or USB-C
cable, follow these steps:

#. :ref:`Open a POS session <pos/session-start>`.

> #. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).

Reviewer: ```suggestion
#. Click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon.
```
Usually, we describe the icon directly after displaying it

> #. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens a new window to drag

Reviewer: ```suggestion
#. Click the :icon:`fa-desktop` (:guilabel:`Customer Display`) icon, which opens a new window to drag
```
I recommend using the tooltip as it is (when there is one obviously :sweat_smile:)

   onto the second screen.

For POS terminals running the Odoo Android app with dual-screen support,

#. :doc:`Activate the Point of Sale Mobile module <../../../general/apps_modules>` to enable the
   customer display.
#. :ref:`Open a POS session <pos/session-start>`.
#. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).
#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens the customer display
   on the terminal's secondary screen.

.. _customer_display/remote:

Remote
------

> To open the customer display on a remote device, (any computer, tablet, or smartphone), follow these

Reviewer: ```suggestion
To open the customer display on a remote device (any computer, tablet, or smartphone), follow these
```

steps:

> #. Access your database from the other device.
> #. Navigate to :menuselection:`Point of Sale --> Dashboard`.

Reviewer: ```suggestion
#. Access your database from the other device and open the :guilabel:`Point of Sale` app.
```
This can be a single step.

> #. Click the :icon:`fa-ellipsis-v` icon (:guilabel:`vertical ellipsis`) on a POS card.
> #. Click :guilabel:`Customer Display` to open the display remotely.

Reviewer: ```suggestion
#. Click the :icon:`fa-ellipsis-v` (:guilabel:`Dropdown menu`) icon on a POS card.
#. Click :guilabel:`Customer Display` to open the display remotely.
```

Reviewer: I usually use the "tooltip" to describe an icon (even if it doesn't describe the icon itself). If there is no tooltip, I describe what it looks like (e.g., vertical ellipsis)

.. note::
   The two devices are not required to share the same network as long as they are connected to the
   same database.

.. _customer_display/iot:

IoT system
----------

To open the customer display onto an IoT-connected screens:

#. :ref:`Open a POS session <pos/session-start>`.

> #. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).
> #. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens the customer display
>    onto the IoT-connected display.

Reviewer: ```suggestion
#. Click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon.
#. Click the :icon:`fa-desktop` (:guilabel:`Customer Display`) icon to open the display
   onto the IoT-connected screen.
```
